internal/constants/constants.go

```diff
@@ -12,6 +12,7 @@ const (
 	ClientPrivateKey   = "tls.key"
 	TrustedCABundleKey = "ca-bundle.crt"
 	Passphrase         = "passphrase"
+	BearerTokenFileKey = "token"
```

Comment:
Please change to BearerTokenKey - the secret contains the actual token data, not a file name, unlike the config flie.
Not a big deal but it causes confusion sometimes.

Reply:
We just point to the file path. It doesn't contain the actual token.
The secret will look like:
`token=/var/run/secrets/kubernetes.io/serviceaccount/token`
Fluentd loki plugin accepts the file path rather than actual token